gating/check/pre

```diff
@@ -24,3 +26,27 @@ fi
 # yaml library are present for ansible to work.
 source "$(readlink -f $(dirname ${0}))/../gating_prerequisites.sh"
 
+# Install pre-requisites for the different types of tests
```

Comment:
Bash arrays could probably be used here.

Reply:
Sure, they could. This works, and is pretty simple though. If you feel strongly about it I can change it up.

Reply:
ya, this was a VERY minor nit